### ISSUE TITLE
border on iron-input

### ISSUE THREAD

**Issue by ratracegrad** [Collaborne/paper-search#30]
Is there a way to override the styling of the iron-input in the paper-search-bar so that it has a 1px border around it? I would like to have the border around the iron-input and not the input inside it. I would like for it to look like this.
![screen shot 2017-11-19 at 4 26 10 pm](https://user-images.githubusercontent.com/10262857/32995814-ed421c38-cd47-11e7-9b02-f701347e3623.png)

**Comment by ronnyroeller** [comment]
@ratracegrad, I don't think you can today but it sounds like a good idea. Feel free to provide a PR that adds a mixin to the iron-input.
